app/src/main/java/com/example/androidpractice2020/InfoFilmActivity.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.androidpractice2020
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.film_info.*
+
+class InfoFilmActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.film_info)
+        val id = intent.extras?.getInt("id")
+        val film = id?.let { FilmRepository().find(it) }
```

Comment:
FilmRepository().find(it)?also {
imageView.setImageResource(it.imageView)
....
}